### ISSUE TITLE
timer IDs aren't spec compliant

### ISSUE THREAD

**Issue by AlexandreBonaventure** [chrisguttandin/worker-timers#382]
**What is expected ?**
According to spec, if I understand correctly, IDs returned by setInterval/setTimeout should be greater than zero. https://html.spec.whatwg.org/multipage/timers-and-user-prompts.html#timers

**What actually happens ?**
`workerTimers.setInterval(cb) // return 0 the first time`

This is actually a big discrepancy because (in my knowledge) it is a common practice to clear the interval/timeouts by doing: 
```js
function startTimer() {
  if (id) clearTimeout(id)
  setTimeout(cd, 1000)
}
```
This snippet would fail with your implementation but works with `window.setTimeout`

**Comment by chrisguttandin** [comment]
Hi @AlexandreBonaventure, thanks for making me aware of that discrepancy. The issue should be fixed in version 6.0.2.

**Comment by AlexandreBonaventure** [comment]
that was quick! thanks
